pom.xml

```diff
@@ -8,9 +8,9 @@
   </parent>
 
   <properties>
-    <jenkins.version>2.150.2</jenkins.version>
+    <jenkins.version>2.190</jenkins.version>
     <java.level>8</java.level>
-    <waffle.version>1.9.0</waffle.version>
+    <waffle.version>2.1.0</waffle.version>
```

Comment:
This version jump and its associated dependency on jna/jna-platform 5.4.0 leave me with some compatibility concerns since Jenkins still depends on 5.3.1